docs/docs/debugging-replace-renderer-api.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Debugging replaceRenderer API
+---
+
+## What is the `replaceRenderer` API?
+
+The `replaceRenderer` API is one of [Gatsby's Server Side Rendering (SSR) hooks](https://www.gatsbyjs.org/docs/ssr-apis/#replaceRenderer). It's used to customise how Gatsby renders your static content. It can be implemented by any Gatsby plugin or your `gatsby-ssr.js` file - adding support for Redux, CSS-in-JS libraries or any code that needs to change Gatsby's default HTML output.
```

Comment:
do a relative link so gatsby-plugin-catch-links can do its thing

Reply:
I'd also immediately make it clear this stuff is all involved when running `gatsby build`. E.g. the second sentence is "This API is called when you run `gatsby build`"